### DISCUSSION TITLE
Transcriptions lost

### DISCUSSION THREAD

**Discussion by pbussi** [deepgram/community#315] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Recently I am getting "Lost" responses using the deepgram API. According to the log:
"The metadata about this response appears to be lost. You won't be charged for this request. If you see this occur repeatedly please contact Deepgram support." 
Can you check this?
Some req ids:
f04c20e2-5548-4316-ae22-3c26c291856b
aef2270d-58f4-429a-a4cd-d86f295f607e
0937e9ad-52b6-43aa-aa6f-987fed6e789a
466c2a7c-ffea-4f1e-98a0-ac40e78a0039

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

f04c20e2-5548-4316-ae22-3c26c291856b

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @pbussi, I'm sorry that the API is not returning results as expected. I looked into these requests and I see that they are indeed not completing. When a request fails to return a response, the best way for you to handle it is by retrying the request. As noted in the log message, you will only be charged for successes.

I also see that you're using whisper-large with English. Whisper is a much more compute-intensive model than Deepgram's in-house models, and it is much more prone to a variety of issues including timeouts, lack of compute resources, and rate-limiting. I highly recommend that you try our Nova model instead, which is faster, more reliable, and more cost-effective. If you have already tried Nova and found that you still prefer Whisper, I'd love to hear any pain points you encountered that are preventing you from switching over.
